docs/source/reference/series.rst

```diff
@@ -25,6 +25,11 @@ Attributes
 
    Series.dtype
 
+.. autosummary::
+   :toctree:  api/
+
+   Series.str
```

Comment:
instead of listing just the str function, we should do it similar to how pandas organizes it. pandas just shows the list all str.xxx functions in the same page as other Series functions.

Reply:
The thing that pandas does is:
https://github.com/pandas-dev/pandas/blob/master/doc/source/conf.py#L481

It uses lots of sphinx magic to accomplish this. Maybe can we keep this out the scope of this PR? I think it's actually pretty tricky to figure this out.

I think we can just keep make the doc show sth like:

```
StringMethods.capticalize()
StringMethods.substr(...)
```
For this PR?